package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "nyc mocha test/*.test.js",
+    "test:alignment": "standard | snazzy && tap --no-esm -J tests/*.js ",
```

Comment:
I am not sure that we should support both frameworks here, specially due to the few amount of code that is added. Could you please provide the tests using mocha? I can also support here if needed.

Reply:
I don't intend to support both frameworks. I'm keeping both during the process of porting and intend to remove one before we call it a new release. When all functionality is ported I will port all tests to one framework.

The `fastify-reply-from` module has a quite larger test set than this module. To me it looks like its faster to get this module to run against the tests in `fastify-reply-from` but then it implies that this module uses `tap` as test framework. Rewriting the tests in `fastify-reply-from` to `mocha` requires a bit more work. Personally I prefer `tap` over `mocha` but I feel I am in no position to rule on what test framework this module should use. I would prefer if you and @mcollina agreed on what to land on.

Reply:
Totally understand your pain, as I have been there rewriting those tests to mocha. The existing `tap` based tests in `fastify-reply-from` would require a lot of transformation to make it work with `fast-proxy`. We should not do this again IMHO. 
Again, I am willing to support writing the test coverage if needed. 

Looking forward to @mcollina considerations here.

Reply:
I'm pretty unsure why they were rewritten in mocha in the first place given that the rest of the code is mostly intact. I very much prefer tap (or ava) over mocha because it minimizes the chances of one test interacting with another test. The design of the test suite in fastify-reply-from is to maximize the benefits of this.

Overall, I agree with the the approach of @trygve-lie.

Reply:
I am all fine doing the job on getting all tests onto one test framework if that was unclear :)

Reply:
I guess we don't need to argue about test frameworks. If @trygve-lie can port the tests back using `tap` I am also fine, respect and welcome.

Reply:
Cool. Very nice to have an easy resolution =)